ibis_substrait/tests/compiler/snapshots/test_tpch/test_compile/tpc_h092/tpc_h092.json

```diff
@@ -1121,7 +1109,7 @@
     }
   ],
   "version": {
-    "minorNumber": 24,
+    "minorNumber": 35,
```

Comment:
The larger Substrait version bump here is because `tpc_h09.json` was unused. Tests relied on `tpc_h090.json` and `tpc_h091.json` instead.